guides/common/modules/proc_adding-custom-deb-repositories.adoc

```diff
@@ -0,0 +1,72 @@
+[id="Adding_Custom_DEB_Repositories_{context}"]
+= Adding {customdebtitle} Repositories
+
+Use this procedure to add {customdeb} repositories in {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-custom-deb-repositories[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
+{Project} automatically completes the *Label* field based on what you have entered for *Name*.
+. Optional: In the *Description* field, enter a description for the repository.
+. From the *Type* list, select `deb` as type of repository.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+You can find the upstream URLs on Debian-based systems in `/etc/apt/sources.list`.
+. Optional: In the *Releases* field, set a or multiple releases.
++
+For official Debian repositories, {Team} recommends setting a _codename_ in the *Releases* field, for example `buster` for Debian 10 or `bullseye` for Debian 11.
+Do not use `stable` or `testing` as they merely reference a currently corresponding _codename_.
+This helps to avoid drastic changes once a new Debian version is released and the reference is changed.
+{Team} recommends creating one repository in {Project} per release to keep things tidy and avoid potential performance and network issues during synchronization.
```

Comment:
We had a conversation about recommends before. 
I realise that it is hard for you to follow these conventions if others don't. Ideally, we would just say
```suggestion
To keep things easy to manage and to avoid potential performance and network issues during synchronization, create one repository in {Project}.
```

Reply:
Applied.